infra/feast-operator/internal/controller/services/services.go

```diff
@@ -860,6 +893,16 @@ func (feast *FeastServices) mountPvcConfig(podSpec *corev1.PodSpec, pvcConfig *f
 	}
 }
 
+func (feast *FeastServices) mountPodVolumes(podSpec *corev1.PodSpec) {
```

Comment:
```suggestion
func (feast *FeastServices) mountServiceVolumes(podSpec *corev1.PodSpec) {
```

Reply:
updated the method name to `mountUserDefinedVolumes`